app/src/main/java/com/bignerdranch/android/rickandmorty/navigation/InfoFragment.kt

```diff
@@ -0,0 +1,89 @@
+package com.bignerdranch.android.rickandmorty.navigation
+
+import android.annotation.SuppressLint
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import coil.load
+import com.bignerdranch.android.rickandmorty.R
+import com.bignerdranch.android.rickandmorty.databinding.FragmentInfoBinding
+import com.bignerdranch.android.rickandmorty.model.PersonInfo
+import com.bignerdranch.android.rickandmorty.retrofit.RickAndMortyService
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import kotlin.properties.Delegates
+
+class InfoFragment : Fragment() {
+
+    private var _binding: FragmentInfoBinding? = null
+    private val binding get() = requireNotNull(_binding)
+    private var load by Delegates.notNull<Call<PersonInfo>>()
```

Comment:
здесь лучше все-таки через, у тебя изначально нет загрузки никакой, следовательно отменять ничего не нужно
```suggestion
    private var load: Call<PersonInfo>? = null
```